python/seldon_core/seldon_client.py

```diff
@@ -1717,6 +1911,135 @@ def grpc_predict_gateway(
         return SeldonClientPrediction(request, None, False, str(e))
 
 
+def rest_feedback_seldon(
+    prediction_request: prediction_pb2.SeldonMessage = None,
+    prediction_response: prediction_pb2.SeldonMessage = None,
+    reward: float = 0,
+    namespace: str = None,
+    seldon_rest_endpoint: str = "localhost:8002",
+    client_return_type: str = "proto",
+    **kwargs,
+) -> SeldonClientFeedback:
+    """
+    Send Feedback to Seldon API Gateway using REST
+
+    Parameters
+    ----------
+    prediction_request
+       Previous prediction request
+    prediction_response
+       Previous prediction response
+    reward
+       A reward to send in feedback
+    namespace
+       k8s namespace of running deployment
+    seldon_rest_endpoint
+       Endpoint of REST endpoint
+    client_return_type
+        the return type of all functions can be either dict or proto
+    kwargs
+
+    Returns
+    -------
+
+    """
+    request = prediction_pb2.Feedback(
+        request=prediction_request, response=prediction_response, reward=reward
+    )
+    payload = feedback_to_json(request)
+    response_raw = requests.post(
+        "http://" + seldon_rest_endpoint + "/api/v1.0/feedback", json=payload,
+    )
+    if response_raw.status_code == 200:
+        success = True
+        msg = ""
+    else:
+        success = False
+        msg = str(response_raw.status_code) + ":" + response_raw.reason
+    try:
+        if len(response_raw.text) > 0:
+            try:
+                if client_return_type == "proto":
+                    response = json_to_seldon_message(response_raw.json())
+                elif client_return_type == "dict":
+                    response = response_raw.json()
+                else:
+                    raise SeldonClientException("Invalid client_return_type")
+            except:
+                response = None
+        else:
+            response = None
+        return SeldonClientFeedback(request, response, success, msg)
+    except Exception as e:
+        return SeldonClientFeedback(request, None, False, str(e))
+
+
+def grpc_feedback_seldon(
```

Comment:
we need to include `raw_request` like in `grpc_feedback_gateway`

Reply:
Got it, let me add that